examples/plotting/histogram.py

```diff
@@ -11,14 +11,15 @@
 
 from bokeh.layouts import gridplot
 from bokeh.plotting import figure, show
-
+from bokeh.palettes import HighContrast3
```

Comment:
@robinokwanma The CI failing because of an`isort` issue: https://github.com/bokeh/bokeh/actions/runs/4522730079/jobs/7965458838?pr=13021#step:6:84

As shown there, you need to move this import of `HighContrast3` right after gridplot and bring back the newline.